File: CountDiv/php/TestCountDiv.php
<?php

require 'CountDiv.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $count_div;

	protected function setup() {
		$this->count_div = new \Codility\CountDiv;
	}

	public function testSolution() {
		$this->assertEquals(3, $this->count_div->solution(6, 11, 2));
	}

}

File: CountDiv/ruby/TestCountDiv.rb
require_relative 'CountDiv'
require 'minitest/autorun'

class Test < MiniTest::Test

	def setup
		@count_div = CountDiv.new
	end

	def test_solution
		assert_equal 3, @count_div.solution(6, 11, 2)
	end
	
end